src/components/Messages/Conversation.js

```diff
@@ -1,53 +1,54 @@
-import React, {useState} from "react";
-import {useSelector} from "react-redux";
-import {Link} from "react-router-dom";
+import React from 'react';
+import {useDispatch} from 'react-redux';
+import {Link} from 'react-router-dom';
+import {findMessagesByConversationThunk} from '../../redux/messageThunks';
 
 /**
  * A component to render each individual conversation.
- * @param conversationFromList conversation from a list of conversations
- * @returns {JSX.Element}
+ * @param conversationconversation from a list of conversations
  */
-const Conversation = ({conversationFromList}) => {
-    const [conversation] = useState(conversationFromList);
-    const userId = useSelector((state) => state.user.data.id);
-    // const dispatch = useDispatch();
+const Conversation = ({conversation}) => {
+    const dispatch = useDispatch();
     const handleDeleteConversation = async (cid) => {
         // TODO
```

Comment:
Todo...?

Reply:
Deleting conversations isn't part of this issue. I retained this stub since I had already written it and will be part of #47